app/src/main/java/com/keylesspalace/tusky/AccountActivity.kt

```diff
@@ -247,13 +241,11 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasSupportF
             override fun onOffsetChanged(appBarLayout: AppBarLayout, verticalOffset: Int) {
 
                 @AttrRes val attribute = if (titleVisibleHeight + verticalOffset < 0) {
-                    accountToolbar.setTitleTextColor(textColorPrimary)
-                    accountToolbar.setSubtitleTextColor(textColorSecondary)
+                    supportActionBar?.setDisplayShowTitleEnabled(true)
 
                     R.attr.account_toolbar_icon_tint_collapsed
                 } else {
-                    accountToolbar.setTitleTextColor(Color.TRANSPARENT)
-                    accountToolbar.setSubtitleTextColor(Color.TRANSPARENT)
+                    supportActionBar?.setDisplayShowTitleEnabled(false)
```

Comment:
the custom emojis don't care about text color, so lets hide the title completely